src/main/java/uk/gov/hmcts/reform/sscs/reference/data/service/PanelCompositionService.java

```diff
@@ -39,11 +39,12 @@ public PanelCompositionService() {
     }
 
     public List<String> getRoleTypes(SscsCaseData caseData) {
-        log.info("PanelCompositionIsEmpty {}", caseData.getPanelMemberComposition().isEmpty());
         if (nonNull(caseData.getPanelMemberComposition()) && !caseData.getPanelMemberComposition().isEmpty()) {
+            log.info("PanelCompositionIsEmpty {}", caseData.getPanelMemberComposition().isEmpty());
```

Comment:
is this needed? it's a bit misleading given that when it goes into this if block it is not empty
we could print it in the else block to work out whether it's null or empty